### ISSUE TITLE
Some formatting is lost when exporting a presentation to `.pdf`

### ISSUE THREAD

**Issue by gy-mate** [slidevjs/slidev#1497]
**Describe the bug**
Some formatting is lost when exporting a presentation with `theme-apple-basic` to `.pdf`:

The public slide show opens the first slide like this:
<img width="400" alt="Screenshot 2024-04-04 at 12 58 53" src="https://github.com/slidevjs/slidev/assets/13014176/8417bb64-bb44-4528-8034-64619ae268cf">

But the exported presentation's first slide looks like this:
<img width="400" alt="Screenshot 2024-04-04 at 12 59 19" src="https://github.com/slidevjs/slidev/assets/13014176/9178b9e1-3b72-41ec-9353-3d7bb4cbcef9">

The second slide should look like this:
<img width="400" alt="Screenshot 2024-04-04 at 13 01 49" src="https://github.com/slidevjs/slidev/assets/13014176/aa2a6556-b928-4da1-b423-6671c204f595">

But it looks like this in the exported file:
<img width="400" alt="Screenshot 2024-04-04 at 13 02 51" src="https://github.com/slidevjs/slidev/assets/13014176/c66621f1-7ce1-4b78-960e-c8833d436858">

Something is going on with font weights and letter spacing.

_I guess this issue belongs here, but it could also go under [`themes`](https://github.com/slidevjs/themes)._

**To reproduce**

1. Clone [this](https://github.com/gy-mate/kalauz) repo
2. `git checkout dev_gymate`
3. `git chekout 80ddf676`
4. `cd presentation`
5. `slidev export` (or `slidev export --dark`, doesn't matter)
6. [This](https://github.com/slidevjs/slidev/files/14865336/slides-export_light.pdf) file is created without any errors

**Desktop**
- OS: macOS `14.4.1`
- Browser: Safari `17.4.1`
- Slidev version: `0.48.8`

**Comment by KermanX** [comment]
It seems that the font `"Helvetica Neue"` isn't loaded in the public slide show, but loaded in the export mode.

The font-family of the text is `"Helvetica Neue",ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji"`

**Comment by kamuiiiii** [comment]
Everything works fine on my Mac machine

**Comment by gy-mate** [comment]
> It seems that the font `"Helvetica Neue"` isn't loaded in the public slide show, but loaded in the export mode.

@KermanX I don't think so. I would guess that the public slide show uses _Helvetica Neue Bold_ and the export uses _Helvetica Neue Regular_ with the font weight increased. (I would rule out the fallback to `ui-sans-serif` or `system-ui`, [which would be](https://blog.jim-nielsen.com/2020/system-fonts-on-the-web/) _SF Pro Display_ or _SF Pro Text_ in my case.) See the comparison below:

<img width="800" alt="Screenshot 2024-04-06 at 16 04 51" src="https://github.com/slidevjs/slidev/assets/13014176/0187c718-aa43-437d-a1ca-3ee89207b5a3">

I don't know which one is the intended display method, but the public slide show looks better to me.

**Comment by gy-mate** [comment]
The macOS versions of Chrome (`123.0.6312.124`) and Microsoft Edge (`123.0.2420.81`) also display the public slide show the same way (incorrectly) as the exported one looks like.

**Comment by KermanX** [comment]
How about using [the `----executable-path` option](https://sli.dev/guide/exporting#executable-path)?

**Comment by nitzanvolman** [comment]
I've encountered a similar issue where the pdf export was exporting incorrect fonts. 
setting `waitUntil: networkidle` and adding `wait: 10000` seem to fix the issue for me.

**Comment by gy-mate** [comment]
Running `slidev export --wait-until 'networkidle' --wait 10000` unfortunately didn't solve the issue for me.

**Comment by micahscopes** [comment]
I'm losing some basic color formatting for text formatted with bold/italic formatting.
